lib/services/local/access_list_test.go

```diff
@@ -403,6 +405,307 @@ func TestAccessListMembersCRUD(t *testing.T) {
+
+	// Delete all access list reviews.
+	err = service.DeleteAllAccessListReviews(ctx, accessList1.GetName())
+	require.NoError(t, err)
+
+	// Verify that access lists reviews are gone.
+	_, _, err = service.ListAccessListReviews(ctx, accessList1.GetName(), 0, "")
+	require.Empty(t, err)
+}
+
+func TestAccessListRequiresEqual(t *testing.T) {
+	tests := []struct {
+		name     string
+		a        accesslist.Requires
+		b        accesslist.Requires
+		expected bool
+	}{
+		{
+			name:     "empty",
+			expected: true,
+		},
+		{
+			name: "both equal",
+			a: accesslist.Requires{
+				Roles: []string{"a", "b", "c"},
+				Traits: trait.Traits{
+					"trait1": []string{"val1", "val2"},
+					"trait2": []string{"val1", "val2"},
+				},
+			},
+			b: accesslist.Requires{
+				Roles: []string{"a", "b", "c"},
+				Traits: trait.Traits{
+					"trait1": []string{"val1", "val2"},
+					"trait2": []string{"val1", "val2"},
+				},
+			},
+			expected: true,
+		},
+		{
+			name: "roles length",
+			a: accesslist.Requires{
+				Roles: []string{"a", "b", "c"},
+			},
+			b: accesslist.Requires{
+				Roles: []string{"a", "b", "c", "d"},
+			},
+			expected: false,
+		},
+		{
+			name: "roles content",
+			a: accesslist.Requires{
+				Roles: []string{"a", "b", "c"},
+			},
+			b: accesslist.Requires{
+				Roles: []string{"a", "b", "d"},
+			},
+			expected: false,
+		},
+		{
+			name: "trait length",
+			a: accesslist.Requires{
+				Traits: trait.Traits{
+					"trait1": []string{"val1", "val2"},
+					"trait2": []string{"val1", "val2"},
+				},
+			},
+			b: accesslist.Requires{
+				Traits: trait.Traits{
+					"trait1": []string{"val1", "val2"},
+					"trait2": []string{"val1", "val2"},
+					"trait3": []string{"val1", "val2"},
+				},
+			},
+			expected: false,
+		},
+		{
+			name: "trait key different",
+			a: accesslist.Requires{
+				Traits: trait.Traits{
+					"trait1": []string{"val1", "val2"},
+					"trait2": []string{"val1", "val2"},
+				},
+			},
+			b: accesslist.Requires{
+				Traits: trait.Traits{
+					"trait1": []string{"val1", "val2"},
+					"trait3": []string{"val1", "val2"},
+				},
+			},
+			expected: false,
+		},
+		{
+			name: "trait values length",
+			a: accesslist.Requires{
+				Traits: trait.Traits{
+					"trait1": []string{"val1", "val2"},
+					"trait2": []string{"val1", "val2"},
+				},
+			},
+			b: accesslist.Requires{
+				Traits: trait.Traits{
+					"trait1": []string{"val1", "val2", "val3"},
+					"trait2": []string{"val1", "val2"},
+				},
+			},
+			expected: false,
+		},
+		{
+			name: "trait values different",
+			a: accesslist.Requires{
+				Traits: trait.Traits{
+					"trait1": []string{"val1", "val2"},
+					"trait2": []string{"val1", "val2"},
+				},
+			},
+			b: accesslist.Requires{
+				Traits: trait.Traits{
+					"trait1": []string{"val1", "val3"},
+					"trait2": []string{"val1", "val2"},
+				},
+			},
+			expected: false,
+		},
+	}
+
+	for _, test := range tests {
+		test := test
```

Comment:
```suggestion
```
nit: not needed without parallel subtests

Reply:
I meant to make this parallel. :-) Thanks!